### PR TITLE
fastly CDN configuration

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
(yes, the api key is revoked *whistle*)

File: .gitmodules [hunk 1]
```diff
@@ -140,3 +140,7 @@
 [submodule "puppet/modules/transition"]
 	path = puppet/modules/transition
 	url = https://github.com/puppetlabs/puppetlabs-transition
+[submodule "ansible/roles/Jimdo.fastly"]
```
Comment (ehelms) [inline .gitmodules]: Why submodule vs. using galaxy to install via requirements?

Comment (evgeni) [reply .gitmodules]: For one because I didn't upload my fork to galaxy, and we use submodules for Puppet too, so it felt in sync.

Comment (ehelms) [reply .gitmodules]: Okie doke. I can live with it, just asking in case there was an opportunity to start clean.

**Pull request merged**